src/extensions/filter-control/utils.js

```diff
@@ -353,6 +353,8 @@ export function createControls (that, header) {
     $.each(header.find('th'), (i, th) => {
       const $th = $(th)
 
+      $th.find('.filter-control').remove()
```

Comment:
Is this needed?

Reply:
Yes, without that we would append the input/selects again, again and again.
Which means we would have X Input/Elements per Header.